packages/material-ui-lab/src/ClockPicker/Clock.tsx

```diff
@@ -235,40 +259,29 @@ function Clock<TDate>(props: ClockProps<TDate> & WithStyles<typeof styles>) {
           </React.Fragment>
         )}
         {numbersElementsArray}
-      </div>
+      </ClockClock>
       {ampm && (wrapperVariant === 'desktop' || ampmInClock) && (
         <React.Fragment>
-          <IconButton
+          <ClockAmButton
             data-mui-test="in-clock-am-btn"
             onClick={() => handleMeridiemChange('am')}
             disabled={meridiemMode === null}
-            className={clsx(classes.amButton, {
-              [classes.meridiemButtonSelected]: meridiemMode === 'am',
-            })}
+            styleProps={styleProps}
           >
             <Typography variant="caption">AM</Typography>
-          </IconButton>
-          <IconButton
+          </ClockAmButton>
+          <ClockPmButton
             disabled={meridiemMode === null}
             data-mui-test="in-clock-pm-btn"
             onClick={() => handleMeridiemChange('pm')}
-            className={clsx(classes.pmButton, {
-              [classes.meridiemButtonSelected]: meridiemMode === 'pm',
-            })}
+            styleProps={styleProps}
           >
             <Typography variant="caption">PM</Typography>
-          </IconButton>
+          </ClockPmButton>
         </React.Fragment>
       )}
-    </div>
+    </ClockRoot>
   );
 }
 
-Clock.propTypes = {
```

Comment:
We can keep the `propTypes` I guess.

Reply:
by keeping `propTypes`, there is an error in the component that render `Clock`.

![image](https://user-images.githubusercontent.com/18292247/118267397-359ea800-b4e6-11eb-896a-9cb859a808e1.png)

I don't think `propTypes` is needed since interface props already exist with those values.

Reply:
If the component is private and in TS, I assume we don't need `propTypes`?